test/child-process.js

```diff
@@ -28,10 +28,16 @@ module.exports.childProcess = function spawnChildProcess(configFile, cb) {
   if (!existsSync(configFilePath))
     return cb(new Error('ENOENT: configFile ' + configFilePath + ' does not exist'));
 
-  _do_spawn(command_args, cb);
+  if (arguments.length > 2) {
+    for (var i=2; i<arguments.length; i++) {
```

Comment:
It is easier to call:

```javascript
command_args.push(Array.prototype.slice.call(arguments, 2));
```

Reply:
Considering https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#32-leaking-arguments this will leak the arguments and cause the node to fail optimising the function.